### PR TITLE
[Disco][Op] broadcast_from_worker0

### PR DESCRIPTION
**Pull request by LeshengJin**
This pr introduces op broadcast_from_zero, which broadcasts input tensor from worker-0 to all other workers.

cc: @junrushao @jinhongyii @tqchen

### CODE REVIEW

File: src/relax/op/ccl/ccl.h [hunk 1]
```diff
@@ -35,6 +35,9 @@ namespace relax {
 /*! \brief AllReduce. */
 Expr allreduce(Expr data, String op_type);
 
+/*! \brief Broadcast data from worker-0 to all other workers. */
+Expr broadcast_from_zero(Expr data);
```
Comment (junrushao) [inline src/relax/op/ccl/ccl.h:39]: Let’s think of a consistent naming for this primitive across Relax and Disco/NCCL. broadcast_from_worker0 or broadcast_from_zero?

**Review by junrushao — APPROVED** [review]
Overall looks good, and just one minor nit on naming

File: src/relax/op/ccl/ccl.h (hunk 1 above)
Comment (LeshengJin) [reply src/relax/op/ccl/ccl.h:39]: I prefer broadcast_from_worker0, which is more straightforward. But I'm not sure if it's ok to have numbers in function names.

Comment (junrushao) [reply src/relax/op/ccl/ccl.h:39]: It’s fine to have numbers

**Review by jinhongyii — APPROVED** [review]

File: src/relax/op/ccl/ccl.h (hunk 1 above)
Comment (tqchen) [reply src/relax/op/ccl/ccl.h:39]: agree about broadcast_from_worker0

Comment (junrushao) [reply src/relax/op/ccl/ccl.h:39]: cool. @LeshengJin please update the PR to reflect the name consistency and let's get it merged today!

Comment (LeshengJin) [reply src/relax/op/ccl/ccl.h]: Updated!

File: src/runtime/disco/nccl/nccl.cc [hunk 2]
```diff
@@ -109,13 +109,14 @@ NDArray AllReduce(NDArray send, ReduceKind reduce_kind) {
   return recv;
 }
 
-void BroadcastFromZero(NDArray buffer) {
+NDArray BroadcastFromZero(NDArray buffer) {
```
Comment (junrushao) [inline src/runtime/disco/nccl/nccl.cc:112]: I'll later change its name to BroadcastFromWorker0

**Pull request merged**